composer.json

```diff
@@ -22,9 +22,14 @@
     "require": {
         "laravel/framework": "~5.1"
     },
+    "require-dev": {
+        "phpunit/phpunit" : "^5.0||^6.0||^7.0",
+        "orchestra/testbench": "~3.0"
+    },
```

Comment:
Bolted these in as dev deps for fairly obvious reasons - they should only be pulled in when hacking on laravel-code-generator directly.  Have I made the allowed versions too narrow?